tests/expected/function-contract/modifies/havoc_pass.expected

```diff
@@ -1,38 +1,13 @@
-copy_replace.assertion\
+.assertion\
 - Status: SUCCESS\
 - Description: "equality"\
 in function copy_replace
 
 VERIFICATION:- SUCCESSFUL
 
-copy.assigns\
+.assigns\
 - Status: SUCCESS\
 - Description: "Check that var_4 is assignable"\
 in function copy
 
-copy.assigns\
-- Status: SUCCESS\
-- Description: "Check that var_3 is assignable"\
-in function copy\
```

Comment:
Yeah, these are quite brittle. Any chance we can just add `var_\d+` to cover any number?

Reply:
AFAIK the test harnesses do not support regexes. They are just substring matches :/

Reply:
The truth is that we shouldn't have temporaries in user messages.